docs/reference/settings-reference.md

```diff
@@ -1,18 +1,10 @@
 ---
 id: settings-reference
 title: Settings YAML Reference
+sidebar_position: 4
```

Comment:
Remove this line and I think we're ready to merge.

Reply:
Removed this in the latest commit.